_episodes/53.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: episode
+title: "53: Understanding your data with Coginiti and Trino"
+date: 2023-11-16
+tags: trino coginiti client analytics
+youtube_id: "SAq6U4RmEvw"
+wistia_id: "kydg7g4prg"
+sections:
+- title: "Intro"
+  desc: "Episode 53: Understanding your data with Coginiti and Trino"
+  time: 0
+- title: "Trino releases"
+  desc: "431-433"
+  time: 140
+- title: "Feature"
+  desc: "Dynamic catalogs"
+  time: 270
+- title: "Competition"
+  desc: "Share your best SQL routine"
+  time: 380
+- title: "Java 21"
+  desc: "Upgrading Trino"
+  time: 465
+- title: "Concept of the episode"
+  desc: "JDBC driver and Coginiti"
+  time: 580
+- title: "Topic"
+  desc: "Deployment modes and Coginiti editions"
+  time: 1080
+- title: "Demo of the episode"
+  desc: "Coginiti and Trino"
+  time: 1240
+- title: "Concept"
+  desc: "Coginiti script and more features"
+  time: 1600
+- title: "Beyond the demo"
+  desc: "Download and more information"
+  time: 3485
+- title: "Rounding out"
+  desc: "SQL training series and Trino Summit"
+  time: 3600
+---
+
+## Hosts
+
+* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
+  at [Starburst](https://starburst.io)
+* [Manfred Moser](https://www.linkedin.com/in/manfredmoser), Director of
+  Technical Content at [Starburst](https://starburst.io),
+  ([@simpligility](https://twitter.com/simpligility))
+
+## Guests
+
+* [Matthew Mullins](https://www.linkedin.com/in/msmullins/), CTO at
+  [Coginiti](https://www.coginiti.co),
+  ([@mullinsms](https://twitter.com/mullinsms))
+* [Roman Nestertsov](https://www.linkedin.com/in/rnestertsov/), Principle
+  Engineer at [Coginiti](https://www.coginiti.co),
+  ([@nestertsov](https://twitter.com/nestertsov))
+
+## Releases 431-433
+
+[Trino 431](https://trino.io/docs/current/release/release-431.html)
+
+* Support for [SQL routines](https://trino.io/docs/current/routines.html) and
+  `CREATE/DROP FUNCTION`
+* Support for `REPLACE` modifier in `CREATE TABLE`
+* Improved latency for prepared statements in JDBC driver
+
+[Trino 432](https://trino.io/docs/current/release/release-432.html)
+
+* Faster filtering on columns containing long strings in Parquet data.
+* Predicate pushdown for real and double columns in MongoDB.
+* Support for Iceberg REST catalog in the `register_table` and `unregister_table` procedures.
+* Support for BEARER authentication for Nessie catalog.
+
+[Trino 433](https://trino.io/docs/current/release/release-433.html)
+
+* Improved support for Hive schema evolution.
+* Add support for altering table comments in the Glue catalog.
+
+Also note that Trino 433 also includes documentation for `CREATE/DROP CATALOG`.
```

Comment:
Should we clarify when this was added? Cause this is saying we added documentation, but nothing about when it became possible.

Reply:
It came over a few releases and quite a while ago .. so I think we should leave it out and have people just use the latest...